inference-engine/src/mkldnn_plugin/nodes/mkldnn_deconv_node.cpp

```diff
@@ -611,6 +608,17 @@ void MKLDNNDeconvolutionNode::prepareParams() {
     }
 
     createDeconvPrim(desc, srcMemPtr, getParentEdgesAtPort(1)[0]->getMemoryPtr(), dstMemPtr, pAttrLocal, selected_pd->getImplementationType());
+
+    if (std::dynamic_pointer_cast<DeconvExecutorInt8>(execPtr)) {
+        primArgs = {{DNNL_ARG_SRC, getParentEdgesAtPort(0)[0]->getMemory().GetPrimitive()},
+                    {DNNL_ARG_WEIGHTS, internalBlobMemory.front()->GetPrimitive()},
+                    {DNNL_ARG_DST, getChildEdgesAtPort(0)[0]->getMemory().GetPrimitive()}};
+    } else {
+        primArgs = {{DNNL_ARG_DIFF_DST, getParentEdgesAtPort(0)[0]->getMemory().GetPrimitive()},
+                    {DNNL_ARG_WEIGHTS, getParentEdgesAtPort(1)[0]->getMemory().GetPrimitive()},
+                    {DNNL_ARG_DIFF_SRC, getChildEdgesAtPort(0)[0]->getMemory().GetPrimitive()}};
```

Comment:
The same thing about getting memory, it seems we already have all the necessary mem pointers.